resources/public/api/conf/2.0/historic_uk_fhl_annual_submission_retrieve.yaml

```diff
@@ -0,0 +1,93 @@
+summary: Retrieve a Historic FHL UK Property Business Annual Submission
+description: |
+  This endpoint enables you to retrieve the income from adjustments and allowances for a furnished holiday lettings (FHL) UK property business for tax years from 2017-18 to 2021-22. A National Insurance number and tax year must be provided. Use this endpoint only for the specified tax year range; for current tax year submissions, use the Retrieve a UK Property Business Annual Submission endpoint.
+  
+  ### Test data
+  <p>Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.</p>
+  <table>
+      <thead>
+          <tr>
+              <th>Header Value (Gov-Test-Scenario)</th>
+              <th>Scenario</th>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td><p>N/A - DEFAULT</p></td>
+              <td><p>Simulates a success response.</p></td>
+          </tr>
+      </tbody>
+      <tbody>
+          <tr>
+              <td><p>NOT_FOUND</p></td>
+              <td><p>Simulates the scenario where no data is found.</p></td>
+          </tr>
+      </tbody>
+  </table>  
+
+tags:
+  - Historic FHL UK Property Business Annual Submission
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
```

Comment:
the raml version of this doesnt contain an acceptHeader. Not sure if needed

Reply:
It's needed